relay-server/src/actors/project.rs

```diff
@@ -739,12 +740,14 @@ impl Project {
 
     /// Obtains a valid project state and passes it to the sender once ready.
     ///
-    /// This first checks if the state needs to be updated. This is the case if the project state
-    /// has passed its cache timeout. The `no_cache` flag forces an update. This does nothing if an
-    /// update is already running in the background.
+    /// This first checks if the state needs to be updated. This is the case if
+    /// the project state has passed its cache timeout. The `no_cache` flag
+    /// forces an update. This does nothing if an update is already running in
+    /// the background.
     ///
-    /// Independent of updating, _stale_ states are passed to the sender immediately as long as they
-    /// are in the [grace period](Config::project_grace_period).
+    /// Independent of updating, _stale_ states are passed to the sender
+    /// immediately as long as they are in the [grace
+    /// period](Config::project_grace_period).
```

Comment:
Why do we do this, what the reasoning behind making the lines shorter?  It's much harder to read now.

Reply:
Ooops didn't notice. Put the link back together in [31372cf](https://github.com/getsentry/relay/pull/2646/commits/31372cfe23bb8398b6ad99b548ccc80ad08864ea).